pkg/api/controllers/script_controller.go

```diff
@@ -13,7 +14,8 @@ import (
 
 type ScriptResponse struct {
 	ErrorResponse
-	Link string `json:"details,omitempty"`
+	Link string                           `json:"details,omitempty"`
+	Txs  []ledger.CommitTransactionResult `json:"txs,omitempty"`
```

Comment:
ledger.CommitTransactionResult is not supposed to be send on network without processing.
Errors inside the structure are internal and not well formatted in json.
In the best case, errors should be translated to api error codes. See https://github.com/numary/ledger/blob/a4fe5fa2e0507f71ec423277f9f7248a50973748/pkg/api/controllers/transaction_controller.go#L23